packages/odyssey-storybook/src/contributing/LabsComponents.stories.mdx

```diff
@@ -0,0 +1,40 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Contributing/Labs Components" />
+
+# Contibuting Odyssey Labs Components
+
+If you are wanting to add components to Odyssey Labs, read this doc first before creating a PR.
+
+## Requirements
+
+Getting components added to Odyssey requires a few steps to be completed first:
+
+1. Get approval from Odyssey design to see if that component even belongs in Odyssey.
+2. The component should be generic enough to allow reuse by all teams at Okta.
+3. Re-export or wrap your custom component, and provide Storybook documentation.
+4. If necessary, create a custom `ThemeProvider` specific to your component.
```

Comment:
I'm curious when an individual component would require its own `ThemeProvider` - wouldn't we want that to happen at the highest level possible?